scripts/release.ts

```diff
@@ -44,6 +46,35 @@ type GitHub = {
     password?: string;
 };
 
+interface ITaskContext {
```

Comment:
can we use `type` instead of `interface`?. IIRC, we are using `interface`s just when a class implemented it.

Same with `interface ITask`

Reply:
Sure. Any particular reason for the preference?

Reply:
We were mixing `type`s and `interface`s all over the code and we didn't have any criteria to use one or another, so, because interface and class in other languages are related we agree to use `type` if there is no class that implement the interface.